src/services/resource-creator.js

```diff
@@ -5,10 +5,14 @@ const utils = require('../utils/schema');
 
 function ResourceCreator(Model, params) {
   const schema = Interface.Schemas.schemas[utils.getModelName(Model)];
-
```

Comment:
[Code Style] Is there a reason to remove this newline and not the following ones?
I think we should keep this one, if it does not bother you.

Reply:
I don't think it was intentional.